geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -512,12 +512,23 @@ Any security-related (properties that begin with <code class="ph codeph">securit
 <td><em>not set</em></td>
 </tr>
 <tr>
-<td>security-client-accessor</td>
+<td>security-auth-token</td>
```

Comment:
Why did you change "security-client-accessor" to "security-auth-token"? I don't see the latter in the product but the client-accessor one exists